storybook/CMakeLists.txt

```diff
@@ -65,6 +65,13 @@ add_executable(FigmaDecoratorModelTest tests/tst_FigmaDecoratorModel.cpp)
 target_link_libraries(FigmaDecoratorModelTest PRIVATE Qt5::Test ${PROJECT_LIB})
 add_test(NAME FigmaModelTest COMMAND FigmaModelTest)
 
+add_executable(QmlTests qmlTests/main.cpp)
+target_compile_definitions(QmlTests
+                           PRIVATE QML_IMPORT_ROOT="${CMAKE_CURRENT_LIST_DIR}"
+                           PRIVATE -DQUICK_TEST_SOURCE_DIR="${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
This tells the testRunner where to search for tests (recursively looking for tst_*.qml). 
By default it will search in the executable folder and this define tells it to look elsewhere.

It fixes something I noticed in the StatusQ qml tests: It doesn't support shadow build. The qt creator build needs to be configured with workingDirectory in the test root folder in order for the tests to run. This it will flood git with build files.